event/src/unvalidated/signed/mod.rs

```diff
@@ -8,16 +10,19 @@ use ipld_core::ipld::Ipld;
 use iroh_car::{CarHeader, CarWriter};
 use serde::{Deserialize, Serialize};
 use ssi::jwk::Algorithm;
-use std::collections::BTreeMap;
+use std::{collections::BTreeMap, str::FromStr as _};
+
+use self::cacao::Capability;
 
 use super::{cid_from_dag_cbor, cid_from_dag_jose};
 
 /// Materialized signed Event.
 pub struct Event<D> {
     envelope: Envelope,
-    payload: Payload<D>,
     envelope_cid: Cid,
+    payload: Payload<D>,
     payload_cid: Cid,
+    capability: Option<(Cid, Capability)>,
```

Comment:
should we be consistent with the other two and either change this to two fields (capability and capability_cid) or change the envelope and payload fields to tuples?

Reply:
I don't want two Options because they would be always correlated (i.e. both None or both Some). But for the required fields I like the clarity of named fields